app/js/components/LabOrdersList/OrdersTable.jsx

```diff
@@ -96,7 +96,6 @@ function OrdersTable({
         locale={locale}
         manual
         pages={pages}
-        getDataWithFilters={filterThrough}
```

Comment:
This filter was no longer being used on this page since all the filtering is now happening server-side during the search.,